lib/packer/instance.rb

```diff
@@ -10,13 +10,13 @@ class Instance
 
     delegate :compile, to: :compiler
 
-    def initialize(root_path: Pathname('.'), config_path: Pathname('config/packer.yml'))
-      @root_path = root_path
-      @config_path = config_path
+    def initialize(root_path: nil, config_path: nil)
```

Comment:
Do you think it's worth adding a way to explicitly set `env` in the constructor here? And use that preferentially if it's set, same as we do for `root_path` and `config_path`.

Reply:
Would prefer if we enforced proper environment configuration. But this is only because I do not see a use case for it.